enterprise/ssl-impl/src/test/java/io/crate/protocols/postgres/SslReqHandlerTest.java

```diff
@@ -71,7 +70,7 @@ public void testSslReqHandler() {
     }
 
     private static void sendSslRequest(EmbeddedChannel channel) {
-        ByteBuf buffer = releaseLater(Unpooled.buffer());
+        ByteBuf buffer = Unpooled.buffer();
```

Comment:
Yep, that's correct. Just checked that. It doesn't have any other references and thus is released when written to the channel.